src/timecode.rs

```diff
@@ -473,42 +619,39 @@ impl Timecode {
     }
 
     /**
-    Returns the number of feet and frames this timecode represents if it were shot on 35mm
-    4-perf film (16 frames per foot). ex: '5400+13'.
+    Create a feet+frames string of `self`.
 
     # What it is
 
-    On physical film, each foot contains a certain number of frames. For 35mm, 4-perf film (the most
-    common type on Hollywood movies), this number is 16 frames per foot. Feet-And-Frames was often
-    used in place of Keycode to quickly reference a frame in the edit.
-
-    # Where you see it
+    The customary text representation of frame-accurate film footage.
 
-    For the most part, feet + frames has died out as a reference, because digital media is not
-    measured in feet. The most common place it is still used is Studio Sound Departments. Many Sound
-    Mixers and Designers intuitively think in feet + frames, and it is often burned into the
-    reference picture for them.
+    The `rep` parameter selects the film format. 35mm, 4-perf vertical
+    pulldown is by far the most common, but other film gauges and pulldowns
+    are supported. See the [FeetFramesRep] enum for all available formats.
 
-    - Telecine.
-    - Sound turnover reference picture.
-    - Sound turnover change lists.
+    # Where you see it
 
-    # Examples
+    - Film assembly and pull lists, chronologs and camera reports
+    - FLX and ALE telecine log files
+    - Avid film pull lists, cut lists and change lists
 
-    ```rust
```

Comment:
Keep the example code please, and maybe add an example so we show the results for two different formats.